docs/content/preview/explore/query-1-performance/pg-hint-plan.md

```diff
@@ -44,6 +44,11 @@ To enable pg_hint_plan, run the following command:
 yugabyte=# SET pg_hint_plan.enable_hint=ON;
 ```
 
+{{<note title="Enable pg_hint_plan for all sessions">}}
```

Comment:
I don't think this needs to be in a note box. You can probably just mention it as:
To enable `pg_hint_plan` for all sessions , set it in the [--ysql_pg_conf_csv](../../../reference/configuration/yb-tserver/#ysql-pg-conf-csv) YB-TServer flag.

Reply:
Also , back-port include this change to older versions as well.

Reply:
Please re-review.